config/intellij-idea-inspections.xml

```diff
@@ -963,7 +963,7 @@ isolated classes and we cannot put them to separate package as it will affect us
     <inspection_tool class="GroovyWaitCallNotInLoop" enabled="true" level="ERROR" enabled_by_default="true" />
     <inspection_tool class="GroovyWaitWhileNotSynchronized" enabled="true" level="ERROR" enabled_by_default="true" />
     <inspection_tool class="GroovyWhileLoopSpinsOnField" enabled="true" level="ERROR" enabled_by_default="true">
-        <option name="ignoreNonEmtpyLoops" value="false" />
+        <option name="ignoreNonEmptyLoops" value="false" />
```

Comment:
This is actually an upstream error. https://github.com/joewalnes/idea-community/blob/1fa2c45953ed08667da52b1b83d44c56eb83b043/plugins/groovy/src/org/jetbrains/plugins/groovy/codeInspection/threading/GroovyWhileLoopSpinsOnFieldInspection.java#L41

Reply:
please skip xml content completely in this file. 
the only target of spelling is comments in this file